x/dclupgrade/handler_test.go

```diff
@@ -231,6 +231,10 @@ func TestHandler_ApproveUpgrade(t *testing.T) {
 	// check proposed upgrade for being deleted
 	_, isFound := setup.Keeper.GetProposedUpgrade(setup.Ctx, msgProposeUpgrade.Plan.Name)
 	require.False(t, isFound)
+
+	// check upgrade for being added to ApprovedUpgrade store
+	_, isFound = setup.Keeper.GetApprovedUpgrade(setup.Ctx, msgProposeUpgrade.Plan.Name)
+	require.True(t, isFound)
```

Comment:
Think where else in these tests positive and negative checks regarding `ApprovedUpgrade` entity should be added. Also do not forget to verify `ApprovedUpgrade` content.

Reply:
added checks

Reply:
In `TestHandler_ProposeUpgradeWhenSeveralVotesNeeded` add a check that `ApprovedUpgrade` has not been added.

Reply:
added